src/Compiler/AbstractIL/ilreflect.fs

```diff
@@ -265,12 +265,12 @@ type TypeBuilder with
         let t = typB.CreateTypeAndLog()
 
         let m =
-            if box t <> null then
+            if not (isNull (box t)) then
```

Comment:
Likely no need to box here, isNull might already do that (needs checking though).

Reply:
I think isNull is just match x with | null -> true | _ -> false
And I don't have courage to make potential functional changes to a place I don't know well.